deploy/cloud/helm/platform/components/operator/templates/profiling-job-rbac.yaml

```diff
@@ -100,9 +101,39 @@ rules:
 # Pods - needed for listing pods by label selector and getting logs from test deployments
 - apiGroups: [""]
   resources: ["pods"]
-  verbs: ["list", "get"]
+  verbs: ["list", "get", "create", "delete"]
 - apiGroups: [""]
   resources: ["pods/log"]
   verbs: ["get"]
 {{- end }}
-# (Remove the trailing blank line at end of file)
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
WIll this override the other ClusterRole? I was thinking only the ClusterRoleBinding would be outside the else